lib/services/local/sessiontracker_test.go

```diff
@@ -87,3 +89,52 @@ func TestSessionTrackerStorage(t *testing.T) {
 	require.Error(t, err)
 	require.Nil(t, session)
 }
+
+func TestSessionTrackerImplicitExpiry(t *testing.T) {
+	ctx := context.Background()
+	bk, err := memory.New(memory.Config{})
+	require.NoError(t, err)
+
+	id := uuid.New().String()
+	id2 := uuid.New().String()
+	srv, err := NewSessionTrackerService(bk)
+	require.NoError(t, err)
+
+	_, err = srv.CreateSessionTracker(ctx, &proto.CreateSessionTrackerRequest{
+		Namespace:   defaults.Namespace,
+		ID:          id,
+		Type:        types.KindSSHSession,
+		Hostname:    "hostname",
+		ClusterName: "cluster",
+		Login:       "root",
+		Initiator: &types.Participant{
+			ID:   uuid.New().String(),
+			User: "eve",
+			Mode: string(types.SessionPeerMode),
+		},
+		Expires: time.Now().UTC().Add(time.Second),
+	})
+	require.NoError(t, err)
+
+	_, err = srv.CreateSessionTracker(ctx, &proto.CreateSessionTrackerRequest{
+		Namespace:   defaults.Namespace,
+		ID:          id2,
+		Type:        types.KindSSHSession,
+		Hostname:    "hostname",
+		ClusterName: "cluster",
+		Login:       "root",
+		Initiator: &types.Participant{
+			ID:   uuid.New().String(),
+			User: "eve",
+			Mode: string(types.SessionPeerMode),
+		},
+		Expires: time.Now().UTC().Add(24 * time.Hour),
+	})
+	require.NoError(t, err)
+
+	require.Eventually(t, func() bool {
+		sessions, err := srv.GetActiveSessionTrackers(ctx)
+		require.NoError(t, err)
+		return len(sessions) == 1
```

Comment:
Do we expect only one item back because the first version is implicitly expired? Is this time-based?

Could we check for the specific item we are looking for? Say, the `id2` item?

Reply:
Yep, we only check for one because we wait for the other one to expire to make sure it's filtered out. I added verification to make sure it's `id2`.